pkg/arvo/app/soto.hoon

```diff
@@ -0,0 +1,171 @@
+::
+::  Soto: A Dojo relay for Urbit's Landscape interface
+::  Relays sole-effects to subscribers and forwards sole-action pokes
+::
+/-  sole
+/+  *server, *soto
+/=  index
+  /^  octs
+  /;  as-octs:mimes:html
+  /:  /===/app/soto/index
+  /|  /html/
+      /~  ~
+  ==
+/=  tile-js
+  /^  octs
+  /;  as-octs:mimes:html
+  /:  /===/app/soto/js/tile
+  /|  /js/
+      /~  ~
+  ==
+/=  script
+  /^  octs
+  /;  as-octs:mimes:html
+  /:  /===/app/soto/js/index
+  /|  /js/
+      /~  ~
+  ==
+/=  style
+  /^  octs
+  /;  as-octs:mimes:html
+  /:  /===/app/soto/css/index
+  /|  /css/
+      /~  ~
+  ==
+/=  soto-png
+  /^  (map knot @)
+  /:  /===/app/soto/img  /_  /png/
+::
+|%
++$  state
+::  bon: bone from Dojo peer
+::
+  $%  [%0 bon=bone]
+  ==
+::
++$  move  [bone card]
+::
++$  poke
+  $%  [%launch-action [@tas path @t]]
+      [%sole-action sole-action]
+      [%json json]
+  ==
+::
++$  card
+  $%  [%poke wire dock poke]
+      [%peer wire dock path]
+      [%http-response =http-event:http]
+      [%connect wire binding:eyre term]
+      [%diff diff]
+  ==
+::
++$  diff
+  $%  [%json json]
+      [%sole-effect sole-effect]
+  ==
+::
+--
+::
+|_  [bol=bowl:gall sta=state]
```

Comment:
-1 for three letter names, it's generally pretty safe to just `=bowl:gall`, `=wire` etc.

Also you might consider chucking `state` in here without a face, so that `bon` (should be `bone`) is directly accessible (as `bon` instead of `bon.sta`).